### PR TITLE
Add oracles page

### LINKED ISSUE #27: Add new page: Oracles
### Description
Summarizes available oracles

### Category
Lisk tools

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?

This PR resolves #27

### CODE REVIEW

File: docs/lisk-tools/oracles.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# Oracles
+
+[Oracles](https://ethereum.org/en/developers/docs/oracles/) provide offchain data onchain.
+This allows code running on a blockchain to access a wide variety of information.
+
+The following Oracles support the Lisk network already:
+
+## RedStone
+
+[RedStone](https://redstone.finance/) offers flexible Data Feeds for Lending Markets, Perpetuals, Options, Stablecoins, Yield Aggregators and other types of novel DeFi protocols.
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/oracles.md]:
```suggestion
[RedStone](https://redstone.finance/) offers flexible Data Feeds for Lending Markets, Perpetuals, Options, Stablecoins, Yield Aggregators, and other types of novel DeFi protocols. 
```

File: docs/lisk-tools/oracles.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+[Oracles](https://ethereum.org/en/developers/docs/oracles/) provide offchain data onchain.
+This allows code running on a blockchain to access a wide variety of information.
+
+The following Oracles support the Lisk network already:
+
+## RedStone
+
+[RedStone](https://redstone.finance/) offers flexible Data Feeds for Lending Markets, Perpetuals, Options, Stablecoins, Yield Aggregators and other types of novel DeFi protocols. 
+
+Builders can choose how they want to consume the data among 3 dedicated models:
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/oracles.md]:
```suggestion
Builders can choose how they want to consume the data from the following three dedicated models:
```

File: docs/lisk-tools/oracles.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+*   [RedStone Core](https://docs.redstone.finance/docs/smart-contract-devs/get-started/redstone-core) (pull oracle) - less than 10s update time, broad spectrum of feeds, best for most use cases. All [Core Price Feeds](https://app.redstone.finance/#/app/tokens) are available for Lisk.
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/oracles.md:32]: Please add a TODO here to update the **Core Price Feeds** link to the Lisk token once the L2 Token is available. Right now Redstone shows the price for the L1 Lisk token.

```suggestion
*   [RedStone Core](https://docs.redstone.finance/docs/smart-contract-devs/get-started/redstone-core) (pull oracle) offers - a less than 10s update time, a broad spectrum of feeds, which is best for most use cases.
All [Core Price Feeds](https://app.redstone.finance/#/app/tokens) are available for Lisk.
```

File: docs/lisk-tools/oracles.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+*   [RedStone Classic](https://docs.redstone.finance/docs/smart-contract-devs/get-started/redstone-classic) (push oracle) - for protocols designed for the traditional oracle interface, customizable heartbeat and deviation threshold.
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/oracles.md]:
```suggestion
*   [RedStone Classic](https://docs.redstone.finance/docs/smart-contract-devs/get-started/redstone-classic) (push oracle) - for protocols designed for the traditional oracle interface, customizable heartbeat, and deviation threshold.
```

File: docs/lisk-tools/oracles.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+*   [RedStone X](https://docs.redstone.finance/docs/smart-contract-devs/get-started/redstone-x) - specifically for Perps and Options, highest update frequency and front-running protection.
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/oracles.md]:
```suggestion
*   [RedStone X](https://docs.redstone.finance/docs/smart-contract-devs/get-started/redstone-x) - specifically for Perps and Options, highest update frequency, and front-running protection.
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Looking good already!

File: docs/lisk-tools/oracles.md (hunk 3 above)
Comment (Tschakki) [reply docs/lisk-tools/oracles.md:32]: Why adding this TODO? As far as I understand, users can use Redstone on Lisk Sepolia already to get price feeds for all the tokens mentioned under the link.

Comment (TalhaMaliktz) [reply docs/lisk-tools/oracles.md:32]: Currently, it is displaying the [Price of Lisk L1](https://app.redstone.finance/#/app/token/LSK), not Lisk L2, hence the suggestion.

Comment (Tschakki) [reply docs/lisk-tools/oracles.md:32]: yeah but that is correct currently

Comment (TalhaMaliktz) [reply docs/lisk-tools/oracles.md:32]: Ummm, should the link lead to the page that shows the price of the Lisk token, or should it lead to the prices of all the available tokens? what's the intention here?

Comment (Tschakki) [reply docs/lisk-tools/oracles.md:32]: To all price feeds. As far as I understand, they are all available to developers

Comment (TalhaMaliktz) [reply docs/lisk-tools/oracles.md:32]: Ah, then no todo is required! thanks for clarifying. 🙌🏼

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 🎉

**Pull request merged**